package.json

```diff
@@ -74,19 +76,24 @@
     "@types/jest": "23.0.0",
     "@types/react": "16.3.17",
```

Comment:
in order to not cause errors with duplicate typings in projects we should use a non-exact version for typings